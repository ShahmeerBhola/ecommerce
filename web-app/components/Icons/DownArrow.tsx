import React from 'react';
import config from '../../config';

const DownArrow: React.FunctionComponent = () => {
  const { white } = config.colors;
  return (
    <svg width="11" height="6" viewBox="0 0 11 6" fill="none" xmlns="http://www.w3.org/2000/svg">
      <path
        d="M5.5 6L11 0.652175L10.3293 8.95037e-07L5.5 4.69565L0.670731 5.06609e-08L-4.86153e-07 0.652174L5.5 6Z"
        fill={white}
      />
    </svg>
  );
};

export default DownArrow;
