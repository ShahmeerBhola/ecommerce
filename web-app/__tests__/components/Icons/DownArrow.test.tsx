import React from 'react';
import renderer from 'react-test-renderer';
import DownArrow from '../../../components/Icons/DownArrow';

describe('DownArrow component', () => {
  test('it renders properly', () => {
    const tree = renderer.create(<DownArrow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
