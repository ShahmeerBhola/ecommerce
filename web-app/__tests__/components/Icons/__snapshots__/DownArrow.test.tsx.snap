// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DownArrow component it renders properly 1`] = `
<svg
  fill="none"
  height="12"
  viewBox="0 0 21 12"
  width="21"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M10.5 12L0 1.30435L1.28049 0L10.5 9.3913L19.7195 0L21 1.30435L10.5 12Z"
    fill="#F23939"
  />
</svg>
`;
